sonaranalyzer-dotnet/src/SonarAnalyzer.Common/SonarAnalyzer.Common.VS2015.csproj

```diff
@@ -20,8 +20,9 @@
 
   <ItemGroup>
     <PackageReference Include="Google.Protobuf" Version="3.1.0" />
+    <PackageReference Include="Google.Protobuf.Tools" version="3.0.0" />
     <PackageReference Include="Microsoft.CodeAnalysis.Workspaces.Common" Version="1.3.2" />
-    <PackageReference Include="System.Collections.Immutable" Version="1.1.37" />
+    <PackageReference Include="System.Collections.Immutable" Version="1.2.0" />
```

Comment:
Have you run some tests to see whether this is still working? Not only compiling.